src/main/java/org/alliancegenome/curation_api/jobs/executors/LoadFileExecutor.java

```diff
@@ -181,7 +184,7 @@ public void runCleanup(DiseaseAnnotationService service, BulkLoadFileHistory his
 		ph.startProcess("Deletion/deprecation of primary objects " + dataProviderName, curiesToRemove.size());
 		for (String curie : curiesToRemove) {
 			try {
-				String loadDescription = dataProviderName + " association bulk load (" + md5sum + ")";
+				String loadDescription = dataProviderName + " " + manual.getBackendBulkLoadType() + " bulk load (" + bulkLoadFile.getMd5Sum() + ")";
```

Comment:
Just to be consistent you might want to change `bulkLoadFile.getMd5Sum()` to `manual.getMd5Sum()`

Reply:
`BulkManualLoad` doesn't have the `getMd5Sum`() method

Reply:
I'm sorry you are capturing the upper class not just casting it... didn't catch that.